docs/building-and-deploying-the-sample-solution.md

```diff
@@ -0,0 +1,256 @@
+
+- Using the Azure web portal, open the Stream Analytics page.
+
+- On the Stream Analytics page, verify that a Stream Analytics job has been created using the name that you specified when you ran the provisioning script (the name that you entered in response to the *StreamAnalyticsJobName* prompt).
+
+- Click the job, and then in the menu bar click Configure
+
+- On the Configuration page, verify that the storage account is set to the value that you specified when you ran the provisioning script (the name that you entered in response to the *StorageAccountName* prompt).
+
+- In the menu bar click Inputs.
+
+- On the Inputs page, verify that an input named input01 has been created and that the type of the input is Event Hub.
+
+- Click input01.
+
+- On the input01 page, in the General section, verify that:
+
+	-  The Service Bus namespace is set to the value that you specified when you ran the provisioning script (the name that you entered in response to the *ServiceBusNamespace* prompt).
+
+	-  The Event Hub name is set to eventhub01.
+
+	-  The Event Hub policy name is set to ManagePolicy.
+
+	-  The Event Hub consumer group is set to consumergroup01.
+
+- In the Serialization section, verify that the event serialization format is set to JSON.
+
+- Return to the job page, and then in the menu bar click Query
+
+- On the Query page, verify that the following query is specified:
+	`SELECT * FROM input01`
+
+- Return to the job page, and then in the menu bar click Outputs.
+
+- On the Outputs page, verify that an output named output01 has been created and that the output type is set to Blob Storage
+
+- Click output01
+
+- On the output01 page, in the General section, verify that:
+
+	-  The storage account is set to the value that you specified when you ran the provisioning script (the name that you entered in response to the *StorageAccountName* prompt).
+
+	-  The container is set to container01.
+
+	-  The filename prefix is set to fabrikam.
+
+	- In the Serialization section, verify that the event serialization format is set to JSON.
+
+### Configuring HDInsight (Optional)
+
+Perform the following steps to configure a Hadoop cluster by using HDInsight:
+
+- Using the Azure web portal, open the HDInsight page.
+
+- On the command bar, click New.
+
+- In the New dialog box, select Hadoop. Enter a cluster name and admin password of your choice, specify the storage account that you used when you ran the provisioning script, and then click Create HDInsight Cluster.
+
+> **Note:** It can take several minutes to create the cluster.
+
+## Testing and Running the Solution
+
+The following sections describe how to test and run the simulator.
+
+### Running the Unit Tests
+
+The UnitTests project ... (*What do these tests demonstrate? - TBD*)
+
+To run the unit tests, on the Visual Studio menu bar click Test, click Run, and then click All Tests.
+
+All tests should succeed without errors.
```

Comment:
Before running the tests you have to change App.config to add an storage account and account key, if you don´t do this many tests will fail.

I didn´t find it in the portal, but to get the storage account key you can execute 

GetAzureStorageKey -StorageAccountName <<StorageAccountName>>

from an Azure Powershell console.